lib/rucio/client/downloadclient.py

```diff
@@ -1145,7 +1145,7 @@ def _get_sources(self, merged_items, resolve_archives=True):
             if force_scheme:
                 schemes = force_scheme if isinstance(force_scheme, list) else [force_scheme]
             else:
-                schemes = ['davs', 'gsiftp', 'https', 'root', 'srm', 'file']
+                schemes = ['davs', 'gsiftp', 'http', 'https', 'root', 'srm', 'file']
```

Comment:
replace with the import from lib/rucio/rse/protocols/__init__.py ?

Reply:
We should change all of these to a list in https://github.com/rucio/rucio/blob/master/lib/rucio/common/constants.py

Reply:
Wont this be set to `None` anyway as soon as list_replicas resolves this correctly?

Reply:
won't it be better to reflect by default all the schemes reported by listing replicas instead of adding some filtering on top ?

Reply:
It's probably fine to just give no schemes to `list_replicas` as this should resolve all schemes since #1262 was implemented.